src/sedtrails/particle_tracer/time.py

```diff
@@ -31,99 +31,96 @@ class Time:
         Updates the current time by adding a delta in seconds.
     """
 
-    # reference_date should be a str, the class will do the transformation into a numpy.datetime64 object
+    # reference_date and start_time should be a str, 
+    # the class will do the transformation into a numpy.datetime64 object
     reference_date: str = field(default='1970-01-01 00:00:00')
-    start_time: int = 0
-    duration: int = 0
-    time_step: float = 1.0
+    start_time: str = field(default='1970-01-01 00:00:00')
+    duration: str = field(default='3D 2H1M3S')
```

Comment:
I think these defaults are misleading. I used those as an example, but I wasn't expecting they will make it to the code.